src/AccessibilityInsights.SharedUxTests/Settings/ConfigurationModelTests.cs

```diff
@@ -127,7 +127,7 @@ public void LoadFromJSON_FileDoesNotExist_DataIsCorrect()
             Assert.AreEqual(TreeViewMode.Control, config.TreeViewMode);
             Assert.AreEqual("1.1.10", config.Version);
 
-            Assert.AreEqual(39, typeof(ConfigurationModel).GetProperties().Length, "Count of ConfigurationModel properties has changed! Please ensure that you are testing the default value for all properties, then update the expected value");
+            Assert.AreEqual(36, typeof(ConfigurationModel).GetProperties().Length, "Count of ConfigurationModel properties has changed! Please ensure that you are testing the default value for all properties, then update the expected value");
```

Comment:
>36 [](start = 28, length = 2)

This looks fishy to me. The number of items in the dictionary went down by 3, but you're still checking the same number of properties?